news/6369.doc.rst

```diff
@@ -0,0 +1 @@
+Provided information about the --platform and --python-version options of the ``pip download`` command
```

Comment:
We reserve documentation news entry for major changes. I don't think this meets that threshold.

Reply:
Yeah, this category type can just be changed to trivial.